pom.xml

```diff
@@ -0,0 +1,51 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.apache.maven.archetypes</groupId>
+    <artifactId>project</artifactId>
+    <packaging>jar</packaging>
+    <version>1.3</version>
+    <name>project</name>
+    <url>http://maven.apache.org</url>
+    <build>
+        <plugins>
+            <plugin>
+                <artifactId>maven-assembly-plugin</artifactId>
+                <version>3.1.0</version>
```

Comment:
[Good Practice]: keep versioning in properties section, easier to maintain

Example:
```
 <properties>
        <rest-assured.version>3.0.4</rest-assured.version>
        <junit.version>4.12</junit.version>
        <testng.version>6.11</testng.version>
        <nimbus-jose-jwt.version>4.27</nimbus-jose-jwt.version>
        <slf4.version>1.7.21</slf4.version>
        <swagger-request-validator.version>1.2.1</swagger-request-validator.version>
        <aspectj.version>1.8.10</aspectj.version>
        <json-unit.version>1.28.2</json-unit.version>
        <selenide.version>4.8</selenide.version>
        <guava.version>23.0</guava.version>
        <jsoup.version>1.8.1</jsoup.version>
        <xsoup.version>0.3.1</xsoup.version>
        <hibernate-core.version>5.2.11.Final</hibernate-core.version>
        <powermock.version>1.7.1</powermock.version>
    </properties>
```

Reply:
done